### DISCUSSION TITLE
PORT 25 BLOCKED

### DISCUSSION THREAD

**Discussion by Itseric100** [postalserver/postal#2267] [Help with using Postal]
I figured my vps server host blocked port 25 on my server but i do not intend to use the port 25 so i changed the port on my postal to 2525 and also 587, even though i changed the port, my emails are still stuck on pending, not a single one of it is sending out, how do i go about this please?

**Comment by willpower232** [comment]
It is likely that your server provider is blocking the outgoing connections to other servers mail ports so if you want to use Postal, you will need to find another server provider.

**Reply by Itseric100** [reply]
Thank you so much
